darwin/exceptions.py

```diff
@@ -1,11 +1,85 @@
 from cmath import exp
 from pathlib import Path
 from textwrap import dedent
-from typing import List
+from typing import List, Optional
 
 from jsonschema.exceptions import ValidationError as jscValidationError
 
-from darwin.datatypes import AnnotationFileVersion
+from darwin.datatypes import AnnotationFileVersion, KeyValuePairDict, UnknownType
+
+
+class DarwinException(Exception):
```

Comment:
Creates a root exception, from which we can branch all of our exceptions.

Includes ability to encapsulate a parent exception or a list of exceptions from a multi task job